src/common/snippets/tests/include/pass/mlp_tokenization.hpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <common_test_utils/ov_test_utils.hpp>
+
+#include "snippets/pass/tokenization.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+class TokenizeMLPSnippetsTests : public TransformationTestsF {
```

Comment:
Please align all files and classes with tokenization pass name: not just `MLP` but `MLPSeq` or `MLPSeqQuantized`

We are working on another MLP pattern. Would be great if can distinguish them in tests.

Reply:
Done